truss/templates/server.Dockerfile.jinja

```diff
@@ -70,7 +70,7 @@ COPY ./server_requirements.txt server_requirements.txt
 RUN pip install -r server_requirements.txt
 
 COPY ./requirements.txt requirements.txt
-RUN pip install -r requirements.txt
+RUN pip install -r requirements.txt -vvv
```

Comment:
Just curious. What does -vvv do? Can't seem to find documentation about it.

Reply:
Good catch actually, I left it here by mistake. -vvv is for highest level of verbosity. This was very useful in finding the root cause of this issue. It's really verbose so we don't want this here.